### PR TITLE
uploading /MI-10-MEAN/Pratiksha/PratikshaA4_1.html && PratikshaA4_2.html

### PR DESCRIPTION
**Pull request by pratikshaghule**

### CODE REVIEW

File: Pratiksha/html/PratikshaA4_2.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    </style>
+</head>
+<body>
+    <h1>Assignment 4</h1>
+    
+    <p>A test cell with merged cells</p>
+    
+    <table>
+        <tr>
+            <th colspan="2" rowspan="2">
```
Comment (MKadeer17) [inline Pratiksha/html/PratikshaA4_2.html:38]: Here write on rowspan only 
Not colspan ok

File: Pratiksha/html/PratikshaA4_2.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+            </th>
+            <th colspan="2">
+                Average
+            </th>
+            <th colspan="2" rowspan="2">
```
Comment (MKadeer17) [inline Pratiksha/html/PratikshaA4_2.html:44]: Here is same remove colspan 
Add only rowspan here.

**Review by VIP5614 — APPROVED** [review]

File: Pratiksha/html/PratikshaA4_2.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        <tr>
+            <th>
+                height
+            </th>
+            <th>
+                weight
+            </th>
+        </tr>
+        <tr>
+            <th colspan="2">
```
Comment (MKadeer17) [inline Pratiksha/html/PratikshaA4_2.html:57]: Don't write colspan here 
Write simple table heading.

**Review by poonam-khode — APPROVED** [review]

File: Pratiksha/html/PratikshaA4_2.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+                Males
+            </th>
+            <td>
+                1.9
+            </td>
+            <td>
+                0.003
+            </td>
+            <td colspan="2">
```
Comment (MKadeer17) [inline Pratiksha/html/PratikshaA4_2.html:66]: Same here also remove colspan.

File: Pratiksha/html/PratikshaA4_2.html (hunk 1 above)
Comment (VIP5614) [reply Pratiksha/html/PratikshaA4_2.html:38]: this line is correct MKaddeer ji

File: Pratiksha/html/PratikshaA4_2.html [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+                40%
+            </td>
+        </tr>
+        <tr>
+            <th colspan="2">
```
Comment (MKadeer17) [inline Pratiksha/html/PratikshaA4_2.html:71]: Remove colspan here also

File: Pratiksha/html/PratikshaA4_2.html [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+                Females
+            </th>
+            <td>
+                1.9
+            </td>
+            <td>
+                0.003
+            </td>
+            <td colspan="2">
```
Comment (MKadeer17) [inline Pratiksha/html/PratikshaA4_2.html:80]: Here is same remove colspan.

File: Pratiksha/html/PratikshaA4_1.html [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+        }
+        table,td,th{
+           width: 70%;
+
+            align-content: center;
+            
+            border : 2px solid green;
+            border-collapse: collapse;
+            padding: 10px;
+            margin: 10px
```
Comment (MKadeer17) [inline Pratiksha/html/PratikshaA4_1.html:23]: Remove margin property.

**Review by mayank30041995 — APPROVED** [review]

**Review by MKadeer17 — APPROVED** [review]

**Pull request merged**